dbcrossbarlib/src/drivers/postgres_sql/mod.rs

```diff
@@ -52,10 +54,13 @@ impl Locator for PostgresSqlLocator {
         table: &Table,
         if_exists: IfExists,
     ) -> Result<()> {
+        // TODO: We use the existing `table.name` here, but this might produce
+        // odd results if the input table comes from BigQuery or another
+        // database with a very different naming scheme.
+        let pg_create_table =
+            PgCreateTable::from_name_and_columns(table.name.clone(), &table.columns)?;
```

Comment:
it feels like this should be configurable...

Reply:
Yeah, probably, but this code is only run if you've doing schema transformations and writing a fresh `CREATE TABLE` out to disk. So for now, I assume people can fix this by hand.